src/containers/PerfherderGraphContainer/index.jsx

```diff
@@ -10,6 +10,11 @@ const styles = () => ({
         marginLeft: '0.2rem',
         marginBottom: -5,
     },
+      title: {
+        backgroundColor: 'black',
+        padding: '.3em',
```

Comment:
Use `rem` instead of `em`.
IIRC `rem` is based off the global size rather than the size of the containing element.
I believe this is important when increasing/decreasing the size of a page for all elements growing in a similar ratio.